scripts/echo_gitignore.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env sh
+
+echo '#------------------------------------------------------------------------------#'
+echo '# gitignore'
+curl -L -s 'https://www.gitignore.io/api/code,emacs,intellij+all,jabref,latex,libreoffice,linux,macos,microsoftoffice,visualstudiocode,windows'
+echo ''
+echo '#------------------------------------------------------------------------------#'
+echo '# custom'
```

Comment:
## Why custom lines below the autogenerated ones from gitignore.io?

The webpage [`gitignore.io`][gitignore.io] is a handy tool to generate typical gitignore-snippets, which can be used directly in a `.gitignore`-file.
This tool can be used to boost up writing them manually.

As far as I understand the [git-documentation for gitignore][git/docs/gitignore]

> (...) within one level of precedence, the last matching pattern decides the outcome (...)

and as far as I remember, the last line in a `gitignore`-file, that matches a pattern, holds.
Therefore, writing custom lines under the automatically generated lines just means overwriting the auto-generated version.

## Why the script `echo_gitignore.sh`?

Adding custom lines can be done without the script, but

1. I simply don't want to enter the curl-line manually.
  I could write some shell-function `gitignore` and I think, that here is a tool that exactly does this, but
2. this tool/function would lead to execute

  ```zsh
  gitignore code,java,... > .gitignore
  ```

  overwriting the custom lines.
  To prevent this, I have to do several clicks in my IDE or copy-paste my custom-lines.
  Since this is too much overhead for me, I just execute the script

  ```zsh
  ./scripts/echo_gitignore.sh > .gitignore && dos2unix .gitignore
  ```

  and have no overhead anymore.
  Matter of taste, but as developer, I prefer to code repeadingly occuring tasks and somehow, I don't know why, I repeadingly have to slightly change my gitignore-files.

[gitignore.io]: https://www.gitignore.io
[git/docs/gitignore]: https://git-scm.com/docs/gitignore#_description